modules/kinesis/src/main/scala/com.snowplowanalytics.snowplow.enrich.stream/sources/KinesisSource.scala

```diff
@@ -91,17 +110,20 @@ class KinesisSource private (
   config: StreamsConfig,
   kinesisConfig: Kinesis,
   sentryConfig: Option[SentryConfig],
-  provider: AWSCredentialsProvider
+  sourceProvider: AwsCredentialsProvider,
+  sinkProvider: AWSCredentialsProvider
```

Comment:
Is there a specific reason not to migrate both consumer and producer?

Reply:
Good point. I think even if there's a reason - I'd like to have it documented somewhere because it feels a bit strange.